presidio_evaluator/data_generator/faker_extensions/providers.py

```diff
@@ -1,6 +1,8 @@
 import random
```

Comment:
Changes here:
- There was an issue with the hospital provider not being able to access the network and failing the data generation process. Added a try mechanism and catch network exceptions
- Ruff (formatting)